substrate/frame/nomination-pools/test-transfer-stake/src/mock.rs

```diff
@@ -40,29 +40,11 @@ pub(crate) const POOL1_REWARD: AccountId = 20397359637244482196168876781421u128;
 
 #[derive_impl(frame_system::config_preludes::TestDefaultConfig)]
 impl frame_system::Config for Runtime {
-	type BaseCallFilter = frame_support::traits::Everything;
-	type BlockWeights = ();
-	type BlockLength = ();
-	type DbWeight = ();
-	type RuntimeOrigin = RuntimeOrigin;
 	type Nonce = Nonce;
```

Comment:
```suggestion
```

This is actually just set to u32 above :). I suspect a few other pallets are the same